### ISSUE TITLE
We need a Wiki

### ISSUE THREAD

**Issue by EverNife** [CrucibleMC/Grimoire-Mixins-1.7.10#1]
We will need a wiki with details on how this project works and how to build it and also among other things related to the project.

**Comment by spannerman79** [comment]
Agreed.  Also https://github.com/CrucibleMC/Grimoire-Mixins/blob/master/README.md#L97...L106 has duplicated entries.

**Comment by EverNife** [comment]
I will transfer all of that to the wiki, will be more simple.
